src/Shared/Utils.cs

```diff
@@ -0,0 +1,17 @@
+using System;
+
+namespace Shared
+{
+    public class Utils
+    {
+
+        public static decimal ConvertKToThousand(string number)
+        {
+            bool result = number.ToLower().Contains("k");
+
+            if (!result) return decimal.Parse(number);
```

Comment:
These 2 lines seems to be redundant?

Since the replace function wouldn't fail even if there is no "K".